src/coreclr/tools/Common/Internal/Runtime/RuntimeConstants.cs

```diff
@@ -57,4 +57,14 @@ internal static class SpecialDispatchMapSlot
         public const ushort Diamond = 0xFFFE;
         public const ushort Reabstraction = 0xFFFF;
     }
+
+    // Keep in sync with RH\src\rtu\runtimeinstance.cpp
```

Comment:
We only have one copy of this now - no longer need to keep in sync with anything.